\doxysection{pandas.\+errors.\+Num\+Expr\+Clobbering\+Error Class Reference}
\hypertarget{classpandas_1_1errors_1_1NumExprClobberingError}{}\label{classpandas_1_1errors_1_1NumExprClobberingError}\index{pandas.errors.NumExprClobberingError@{pandas.errors.NumExprClobberingError}}


Inheritance diagram for pandas.\+errors.\+Num\+Expr\+Clobbering\+Error\+:
% FIG 0


Collaboration diagram for pandas.\+errors.\+Num\+Expr\+Clobbering\+Error\+:
% FIG 1


\doxysubsection{Detailed Description}
\begin{DoxyVerb}Exception raised when trying to use a built-in numexpr name as a variable name.

``eval`` or ``query`` will throw the error if the engine is set
to 'numexpr'. 'numexpr' is the default engine value for these methods if the
numexpr package is installed.

Examples
--------
>>> df = pd.DataFrame({'abs': [1, 1, 1]})
>>> df.query("abs > 2") # doctest: +SKIP
... # NumExprClobberingError: Variables in expression "(abs) > (2)" overlap...
>>> sin, a = 1, 2
>>> pd.eval("sin + a", engine='numexpr') # doctest: +SKIP
... # NumExprClobberingError: Variables in expression "(sin) + (a)" overlap...
\end{DoxyVerb}
 

Definition at line \mbox{\hyperlink{pandas_2errors_2____init_____8py_source_l00553}{553}} of file \mbox{\hyperlink{pandas_2errors_2____init_____8py_source}{\+\_\+\+\_\+init\+\_\+\+\_\+.\+py}}.



The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
/home/jam/\+Research/\+IRES-\/2025/dev/src/llm-\/scripts/testing/hypothesis-\/testing/hyp-\/env/lib/python3.\+12/site-\/packages/pandas/errors/\+\_\+\+\_\+init\+\_\+\+\_\+.\+py\end{DoxyCompactItemize}
